applicationconfig/client/README.md

```diff
@@ -0,0 +1,87 @@
+# Azure App Configuration client library for Java
+Azure App Configuration provides a service to centrally manage application settings. Modern programs, especially programs running in a cloud, generally have many components that are distributed in nature. Spreading configuration settings across these components can lead to hard-to-troubleshoot errors during an application deployment. Use App Configuration to store all the settings for your application and secure their accesses in one place.
+
+Use the client library for App Configuration to:
+- Add a new Configuration Setting
+- Update an existing Configuration Setting
+- Retrieve a Configuration Setting
+- Delete a Configuration Setting
+
+[Source code]() | [Package (Maven)]() | [API reference documentation]() | [Product documentation](https://docs.microsoft.com/en-us/azure/azure-app-configuration/)
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)](https://docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable) with version 8 or above
+- [Apache Maven](https://maven.apache.org/) 3.0 or above
+- Azure Service Principal
```

Comment:
Do we need a service principal to use this SDK? I thought we just used a connection string for the service.

Reply:
Removed